src/components/shared/Svg.css

```diff
@@ -45,7 +45,6 @@
 
 .arrow svg {
   fill: var(--theme-splitter-color);
-  margin-top: 3px;
```

Comment:
we need to keep this as the default for arrows

we can add this style in sources.css
```diff
.sources-list .tree .arrow svg {
  margin-top: 0px;
}

```